openapi/openapi.yaml

```
@@ -1506,7 +1506,7 @@ paths:
         - name: index_url
           in: query
           required: true
-          description: A secret to register the given Python package index.
+          description: URL to the Python simple repository as described in PEP 503.
```

```suggestion
          description: URL to the Python package index to register.
```
Here `https://pypi.org/simple` is the default value

---

It's not on the register endpoint though, but in the state one (enabling/disabling the index); I reused the description from the register endpoint (in the request body, `properties.url`)

So maybe `package index to enable or disable` ?

---

Correct :+1: